sdcm/nemesis.py

```diff
@@ -3039,10 +3039,19 @@ def _restore_schema(locations: list, cluster_id: str, tag: str) -> None:
         stress_queue = execute_data_validation_thread(command_template=confirmation_stress_template,
                                                       keyspace_name=chosen_snapshot_info["keyspace_name"],
                                                       number_of_rows=chosen_snapshot_info["number_of_rows"])
-        for stress in stress_queue:
-            is_passed = self.tester.verify_stress_thread(stress)
-            assert is_passed, (
-                "Data verification stress command, triggered by the 'mgmt_restore' nemesis, has failed")
+        try:
+            for stress in stress_queue:
+                is_passed = self.tester.verify_stress_thread(stress)
+                assert is_passed, (
+                    "Data verification stress command, triggered by the 'mgmt_restore' nemesis, has failed")
+        finally:
```

Comment:
Why do we have it in finally for stress? The drop is not tied to the stress, it can be simply appended after the stress section, right?